packages/anasazi/epetra/test/LOBPCG/cxx_main.cpp

```diff
@@ -190,11 +196,13 @@ int main(int argc, char *argv[])
   MyPL.set( "Use Locking", true );
   MyPL.set( "Locking Tolerance", tol/10 );
   MyPL.set( "Full Ortho", fullOrtho );
+  MyPL.set( "Orthogonalization", ortho );
+
   if (testRecovery) {
     // initialize with bad P
     RCP<Anasazi::LOBPCGState<ScalarType,MV> > badstate = rcp( new Anasazi::LOBPCGState<ScalarType,MV>() );
-    Anasazi::SVQBOrthoManager<ScalarType,MV,OP> ortho(M);
-    ortho.normalize(*ivec,null);
+    Anasazi::SVQBOrthoManager<ScalarType,MV,OP> orthoM(M);
```

Comment:
If users can specify the orthogonalization method, then why is SVQBOrthoManager hard-coded here?  I'm guessing that this a special case and not the main orthogonalization loop in LOBPCG.

Reply:
This is a special case where we are trying to create a bad subspace to make sure the recovery mechanism in the solver works.  The orthogonalization method is not critical to this part of the test.

Reply:
Cool; got it!